skywalking/config.py

```diff
@@ -59,13 +59,14 @@
 kafka_bootstrap_servers = os.getenv('SW_KAFKA_REPORTER_BOOTSTRAP_SERVERS') or "localhost:9092"  # type: str
 kafka_topic_management = os.getenv('SW_KAFKA_REPORTER_TOPIC_MANAGEMENT') or "skywalking-managements"  # type: str
 kafka_topic_segment = os.getenv('SW_KAFKA_REPORTER_TOPIC_SEGMENT') or "skywalking-segments"  # type: str
+celery_parameters_length = int(os.getenv('SW_CELERY_PARAMETERS_LENGTH') or '512')
 
 
 def init(
         service: str = None,
         instance: str = None,
         collector: str = None,
-        protocol_type: str = 'grpc',
+        protocol_type: str = None,
```

Comment:
I'd rather leave `grpc` as default protocol and switch to `http` in the tests.

Reply:
Grpc IS the default protocol because of config.py:36
```
protocol = (os.getenv('SW_AGENT_PROTOCOL') or 'grpc').lower()  # type: str
```
What that line did was override whatever protocol was set by env var `SW_AGENT_PROTOCOL` with `grpc`, which was a bug.

Reply:
I see